docs/other/running_synapse_on_single_board_computers.md

```diff
@@ -38,8 +38,7 @@ use_presence: false
 # joins a new remote room. If it is above the complexity limit, the server will
 # disallow joining, or will instantly leave.
```

Comment:
The above sounds like you may or may not want to enable it. I think we should reword this to be clearer:

> Disable room "complexity" checking before a user joins a new remote room.
> If it is above the complexity limit, the server will disallow joining, or will
> instantly leave.

Reply:
Ahhh, well that's confusing that it is copied from the sample config when bits and pieces of the values are modified. That's probably why it doesn't make sense.

I think it would be clearer, yes. We're telling people to paste this into their config, so it should describe what that change actually does IMO?

Reply:
I've made the comments more relevant in c30b24717. Hopefully that's better?